src/CircleEditor/CircleEditorException.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Hmm.. Is this file copied from `src/Utils/Balloon.ts`?
I couldn't find any difference except class name T.T

Reply:
Yes the only difference is that exceptionAlert function that we customized uses 'vscode.window.showErrorMessage' and this shows different color from 'showInformationMessage'. Is it better to use info function in Balloon.ts?

Reply:
If so, what is different between `error(msg, false)` and `execptionAlert(msg)`...? @.@

Reply:
exceptionAlert does not show log information.

Reply:
I will change this part to use Balloon.ts and just give showSeeLogBtn option 'false' in error function. Thank you!!